resources/settings.xml

```diff
@@ -3,6 +3,9 @@
     <setting id="search_signed" label="30110" type="bool" default="false" />
     <setting id="search_ad" label="30120" type="bool" default="false" />
     <setting id="subtitles" label="30130" type="bool" default="false" />
+    <setting label="Enable BBC iD" default="false" type="bool" id="bbc_id_enabled"/>
+    <setting label="BBC Username" default="" type="text" id="bbc_id_username" enable="eq(-1,true)" subsetting="true"/>
+    <setting label="BBC Password" default="" type="text" id="bbc_id_password" enable="eq(-2,true) + !eq(-1,)" subsetting="true" option="hidden"/>
```

Comment:
Done already for next commit, I was probably a little to keen and eager to try and share last night. :) I also need to work out when is best to do the login, or use existing cookies. Then I will disable/hide the Watching and Favourites menus if not logged in. 

Do you know where I could show a login status?